$NetBSD: patch-CMakeLists.txt,v 1.1 2019/10/01 12:28:19 nia Exp $

Don't allow it to default to /usr/share/games/quake2.

--- CMakeLists.txt.orig	2019-09-26 13:14:20.599841947 +0000
+++ CMakeLists.txt
@@ -81,6 +81,7 @@ add_definitions(-DYQ2ARCH="${ARCH}")
 # Systemwide installation of game assets.
 if(${SYSTEMWIDE_SUPPORT})
     add_definitions(-DSYSTEMWIDE)
+    add_definitions(-DSYSTEMDIR="${CMAKE_INSTALL_PREFIX}/share/yquake2")
 endif()
 
 # We need to pass some options to minizip / unzip.
